grug/app/src/app.rs

```diff
@@ -46,9 +47,26 @@ pub struct App<DB, VM, PP = NaiveProposalPreparer> {
     /// Related config in CosmWasm:
     /// <https://github.com/CosmWasm/wasmd/blob/v0.51.0/x/wasm/types/types.go#L322-L323>
     query_gas_limit: u64,
+    #[cfg(feature = "indexer")]
+    pub indexer_app: INDEXER,
+    #[cfg(feature = "indexer")]
+    pub indexing_enabled: bool,
```

Comment:
Without the feature enabled this code won't compile since the generic isn't used anywhere.